Electron/src/RON/components/PingGraph.tsx

```diff
@@ -58,51 +61,37 @@ class PingGraph extends Component<IProps, IState> {
     this.state = {
       ping: [],
       board: "Drive",
-      interval: setInterval(() => this.ICMP("Drive"), 1000),
+      interval: setInterval(() => this.update("Drive"), 1000),
```

Comment:
are you planning on having drive update less frequently? would that help any of the NaN problems we've been having?

Reply:
This is actually the file that we want updating the most frequently. We want this graph to show an analysis of what has been happening recently for the single board we care about the most at the moment. If anything, we may want to decouple this update from ping tool, as ping tool and ping map may only need to update once every ~10 seconds while we want this one to be more frequent.